src/components/Notifications/DataAlerts/DataAlerts.js

```diff
@@ -346,24 +355,27 @@ export default class DataAlerts extends React.Component {
                       )}
                       {this.hasError(notification) ? (
                         <React.Fragment>
-                          <Button
-                            type="primary"
-                            tooltip="This Alert is no longer active. <br /> Click to re-initialze it."
-                            multiline
-                            className="react-autoql-re-initialize-btn"
-                            onClick={() => {
-                              this.props.onAlertInitializationCallback(
-                                notification,
-                                this.props.selectedDemoProjectId,
-                                this.props.authentication
-                              )
-                              this.getDataAlerts()
-                            }}
-                          >
-                            <span className="react-autoql-re-initialize-btn-text">
-                              <Icon type="warning-triangle" /> Resend
-                            </span>
-                          </Button>
+                          {notification.type === 'CUSTOM' ? (
```

Comment:
Instead of 
`{notification.type === 'CUSTOM' ? (...) : (null)}`
you could also do
`{notification.type === 'CUSTOM' && (...)}`